src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/AnalyzePackageStatus.cs

```diff
@@ -50,7 +59,7 @@ public async Task AnalyzeAsync(AnalyzeContext analysis, CancellationToken token)
                     Logger.LogCritical(exc, "Unexpected exception analyzing package references for: {ProjectPath}", context.CurrentProject.Required().FileInfo);
                 }
 
-                Logger.LogInformation("Package Analysis for {ProjectPath}", project.FileInfo.Name);
+                Logger.LogInformation("Package Analysis for {ProjectPath} for the target TFM of {TargetTFM}", new object[] { project.FileInfo.Name, targetTfm });
```

Comment:
You don't need to pass it as an array (ie there's a params overload)

Reply:
there isnt one - https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerextensions.loginformation?view=dotnet-plat-ext-5.0 - I can only see the object[] as parameter options under Microsoft.Extensions.Logging

Reply:
The one @twsouthwick is thinking of is an extension method: https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LoggerExtensions.cs#L172

This can become: `Logger.LogInformation("Package Analysis for {ProjectPath} for the target TFM of {TargetTFM}", project.FileInfo.Name, targetTfm);`

Reply:
changed